modules/gltf/src/lib/extensions/KHR_techniques_webgl.js

```diff
@@ -0,0 +1,58 @@
+// GLTF EXTENSION: KHR_techniques_webgl
+// https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_techniques_webgl
+/* global TextDecoder */
+import GLTFScenegraph from '../gltf-scenegraph';
+import {KHR_TECHNIQUES_WEBGL} from '../gltf-constants';
+
+function resolveTechniques({programs = [], shaders = [], techniques = []}, gltfScenegraph) {
+  const textDecoder = new TextDecoder();
+
+  shaders.forEach(shader => {
+    if (Number.isFinite(shader.bufferView)) {
+      shader.code = textDecoder.decode(
+        gltfScenegraph.getTypedArrayForBufferView(shader.bufferView)
+      );
+    }
+
+    // TODO: handle URI shader
```

Comment:
Would it make sense to issue a warning here? It is painful when things don't load and we have to trace it back to something like this.